client/components/Main.jsx

```diff
@@ -61,6 +61,10 @@ export default class Main extends React.Component {
       settingsDialogOpened: false,
       // Snackbar message
       showMessage: false,
+      filterRegexProps: {
+        active: true,
```

Comment:
Please have this as false per default. User might be confused why this is on and what it does